src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -4136,10 +4136,10 @@ string
 **children**
 
 Function that will be called when each value is rendered. Defaults to \`function children(value) {
-    return _react[\\"default\\"].createElement(_Box.Box, {
+    return _react.default.createElement(_Box.Box, {
```

Comment:
hm, why do we have these changes?

Reply:
This was an error, I just removed these now from changes. I wonder how I got there.